Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM ubuntu:noble
+RUN apt-get update && apt-get install -y ca-certificates wget unzip git make && update-ca-certificates
+
+RUN mkdir -p ~/.eval-dev-quality
```

Comment:
Please use an "eval" user here. So we do not run as root. that helps with lots of isolation problems